### PR TITLE
Creates `XrpEscrowCancel` Protobuf Wrapper Class

### PR DESCRIPTION
**Pull request by tedkalaw**
## High Level Overview of Change
This PR implements the class XRPEscrowCancel - a native Typescript class to wrap the EscrowCancel protocol buffer. See also https://xrpl.org/escrowcancel.html

### Context of Change
Effort to implement additional XRPL transaction types beyond Payment in the SDK.

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [X] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

## Before / After
XRPEscrowCancel class and conversion unit tests now exist.

## Test Plan
Unit tests added to verify conversion. CI passes.

### CODE REVIEW

File: src/main/java/io/xpring/xrpl/model/XrpEscrowCancel.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  static XrpEscrowCancel from(EscrowCancel escrowCancel, XrplNetwork xrplNetwork) {
+    if (!escrowCancel.hasOfferSequence() || !escrowCancel.hasOwner()) {
+      return null;
+    }
+
+    final Integer offerSequence = escrowCancel.getOfferSequence().getValue();
+
+    ClassicAddress ownerClassicAddress = ImmutableClassicAddress.builder()
+        .address(escrowCancel.getOwner().getValue().getAddress())
+        .isTest(xrplNetwork == XrplNetwork.TEST || xrplNetwork == XrplNetwork.DEV)
```
Comment (keefertaylor) [inline src/main/java/io/xpring/xrpl/model/XrpEscrowCancel.java:59]: optional: You'll time this a lot so we may consider making a helper method on XrpUtils for it.

File: src/main/java/io/xpring/xrpl/model/XrpEscrowCancel.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+   *         to cancel.
+   */
+  Integer offerSequence();
+
+  /**
+   * Address of the source account that funded the escrow payment, encoded as an X-address
+   * (see https://xrpaddress.info/).
+   *
+   * @return A {@link String} containing the address of the source account that funded the escrow payment, encoded as
+   *         an X-address
```
Comment (amiecorso) [inline src/main/java/io/xpring/xrpl/model/XrpEscrowCancel.java:37]:
```suggestion
   *         an X-address.
```

Comment (tedkalaw) [reply src/main/java/io/xpring/xrpl/model/XrpEscrowCancel.java]: got it

File: src/main/java/io/xpring/xrpl/model/XrpEscrowCancel.java (hunk 1 above)
Comment (tedkalaw) [reply src/main/java/io/xpring/xrpl/model/XrpEscrowCancel.java:59]: incoming next pr

**Pull request merged**